clients/src/main/java/org/apache/kafka/common/security/ssl/SslEngineBuilder.java

```diff
@@ -66,6 +67,7 @@
         this.configs = Collections.unmodifiableMap(configs);
         this.protocol = (String) configs.get(SslConfigs.SSL_PROTOCOL_CONFIG);
         this.provider = (String) configs.get(SslConfigs.SSL_PROVIDER_CONFIG);
+        SecurityConfig.addConfiguredSecurityProviders(this.configs);
```

Comment:
Will the function "addConfiguredSecurityProviders"  be called to install Java providers at the very beginning of starting Java process before any Java classes? There is no need to call the function every class?